itou/openid_connect/inclusion_connect/views.py

```diff
@@ -173,13 +173,15 @@ def inclusion_connect_activate_account(request):
         return HttpResponseRedirect(params.get("previous_url", reverse("search:employers_home")))
 
     user_kind = params.get("user_kind")
-    user = User.objects.filter(email=email).first()
+    email_address = EmailAddress.objects.filter(email__iexact=email).first()
```

Comment:
```suggestion
    email_address = EmailAddress.objects.filter(email__iexact=email).select_related("user").first()
```
Et a priori à chaque fois qu'on utilise `EmailAddress` pour récupérer un `User`.